src/Components/TodoList/TodoListText/TodoListText.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+  Box,
+  Button,
+  FormControl,
+  IconButton,
+  OutlinedInput,
+  TextField,
+} from "@mui/material";
+import React, { useRef, useState } from "react";
+import AddIcon from "@mui/icons-material/Add";
+import { COLORS } from "../../../colors";
+
+export default function TodoListText() {
+  const [description, setDescription] = useState("");
+
+  const handleChange = (event: any) => {
```

Comment:
event type should not be any.
do a quick google search to find out what type onChangeEvent should have.